alarmpy/alarmpy.py

```diff
@@ -5,6 +5,7 @@
 from datetime import datetime
 from pathlib import Path
 from time import sleep, time
+import paho.mqtt.client as mqtt
```

Comment:
If this were to be an optional dependency, I'd wrap this in a try block that catches an `ImportError` and passes